src/commands/utils.js

```diff
@@ -0,0 +1,16 @@
+export function getI18nKindWarning(
+  astWarnings,
+  warnings,
+  getToken = (i18nToken) => i18nToken
+) {
+  return astWarnings.map((warning) => {
+    return {
+      ...warning,
+      kind: getToken(
+        Object.values(warnings).find(
+          (constantWarning) => warning.kind === constantWarning.code
```

Comment:
WDYT about extracting this two lines in a const above?

Reply:
:) good idea